src/styles/index.css

```diff
@@ -0,0 +1,85 @@
+.page {
+  margin: 0;
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+}
+
+.page__clock {
+  position: relative;
+}
+
+.watch {
+  width: 80vmin;
+  height: 80vmin;
+  border-radius: 50%;
+  border: 1vmin dotted #000;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.watch::before {
+  content: '';
+  display: inline-block;
+  width: 1.5vmin;
+  height: 38vmin;
+  background-color: #2c8000;
+  left: 50%;
+  bottom: 50%;
+  position: absolute;
+  z-index: 2;
+  transform: translateX(-50%) rotate(0deg);
+  transform-origin: bottom;
+  animation: clockwork infinite;
+  animation-duration: 60s;
+  animation-timing-function: linear;
+}
```

Comment:
Is there a specific reason for using `vmin` as a unit for width and height? While it's not wrong, it's not a commonly used unit in CSS. Unless you have a specific reason for using it (like responsive design), it might be better to use `px` or `em`.